README.md

```diff
@@ -155,37 +130,15 @@ Users want to determine several types of the formatting width as following
     ]
 ```
 
-#### NOTE (For stage 3)
-
-* We are discussing to use `largestField` and `smallestField` options to determine the time fields instead of an array.
+### Hide zero-value fields
 
-## Hide zero-value fields
+In most cases, users want to avoid displaying zero-value fields. All zero-valued fields are hidden by default. If you specify the style for a specific field, then it is always displayed, but you can override that behavior by also setting the display option for that particular field to `"auto"` again explicitly.
 
-* Users needs to hide zero-value fields depends in some criteria
-  + Hide all zero-value fields
-    - Hours, Minutes, Seconds (without the `hide` option)
-      * 1 hours, 0 minutes and 59 seconds.
-    - Hours, Minutes, Seconds (with the `hide` option)
-      * 1 hours and 59 seconds.
-  + Hide-upper-limit with zero-valued.
-    - Hours, Minutes, Seconds, MilliSeconds (without the `hide-upper-limit` option)
-      * 0 hr, 20 min, 0 sec, 100 msec .
-    - Hours, Minutes, Seconds MilliSeconds(with the `hide-upper-limit` option)
-      * 20 min, 0 sec, 100 msec.
-  + Hide-lower-limit with zero-valued.
+#### Design
 
-### Design
+For each field `foo`, there is an option `fooDisplay` that is set to `"auto"` by default. If you specify the style for that field by setting the `foo` option, then the default for `fooDisplay` becomes `"always"`.
```

Comment:
```suggestion
For each field `foo`, there is an option `fooDisplay` that is set to `"auto"` by default. Setting that option to `"always"` causes that field to be displayed even if it is zero; for example, to always show the "day" field, set `{ dayDisplay: "always" }`. If you specify the style for that field by setting the `foo` option, then the default for `fooDisplay` becomes `"always"`; for example, `{ day: "short" }` implies `{ day: "short", dayDisplay: "always" }`.
```

Reply:
If adding a new sentence, ideally it'd be on a different line for easier diffs later.